Sources/Log.cpp

```diff
@@ -0,0 +1,38 @@
+#include "Log.h"
+
+QFile logs("log-"+ QDateTime::currentDateTime().toString() + "-" + QUuid::createUuid().toString() + ".txt");
+QTextStream logstream(&logs);
```

Comment:
Both `logs` and `logstream` should be initialized in function, not globally (e.g. it should be `setDevice` call for `logstream`).
Also it will be better to place logfile in the `debug_reports` folder.